Makefile

```diff
@@ -1,3 +1,20 @@
+# Copyright © 2020 The OpenEBS Authors
+#
+# This file was originally authored by Rancher Labs
+# under Apache License 2018.
```

Comment:
We can do something like this ?
```
Copyright © 2020 The Rancher Labs and OpenEBS Authors
```

Reply:
The above message looks fine Prateek. As the Rancher Labs authors are currently not in the maintainers list for jiva.